pems-server/src/main/java/org/maroubra/pemsserver/monitoring/MonitoringServiceImpl.java

```diff
@@ -69,6 +70,10 @@ public Completable createSensor(SensorConfig config) {
         return sensorConfigCollection.insertOne(config).toCompletable().doOnCompleted(() -> startSensor(sensor));
     }
 
+    public List<SensorLog> getSensorLogs(String sensorId, int limit) {
+        return sensorLogsCollection.find(Filters.eq("sensorID",sensorId)).limit(limit).toObservable().toList().toBlocking().single();
```

Comment:
I think it should be "sensorId" not "sensorID" (naming in the config class)

Reply:
fixed